src/data/real/liouville.lean

```diff
@@ -117,7 +116,7 @@ begin
     exact mul_le_mul_of_nonneg_left (le_max_right _ M) dist_nonneg }
 end
 
-lemma exists_pos_real_of_irrational_root {α : ℝ} (ha : irrational α)
+lemma exists_pos_real_of_irrational_root {α : ℝ} (ha : _root_.irrational α)
```

Comment:
To avoid the need for the `_root_` here, you should protect the lemma `liouville.irrational` i.e., add `@[protected]`. In this way, it can only be accessed with `liouville.irrational` or `h.irrational` for `h : liouville x`, and no conflict can arise.

Reply:
Thanks for explaining this: I did not know what the attribute `protected` did!